modules/k3s/build.gradle

```diff
@@ -0,0 +1,12 @@
+description = "Testcontainers :: K3S"
+
+dependencies {
+    api project(":testcontainers")
+
+    shaded 'com.fasterxml.jackson.core:jackson-databind:2.13.1'
```

Comment:
Just realized - do we really need it? `core` module already shades `jackson-databind`
```suggestion
```

Reply:
I thought about it quite a bit, but wanted to be explicit about the dependency as it's used directly.

Pleasingly the resulting JAR for this module _does not_ include the shaded `jackson-databind` classes again, but _does_ include shaded `jackson-dataformat-yaml` classes which are not already shaded into `core`.

In my eyes I think this feels right.

Reply:
Interesting, I guess this is a side effect of our "automatic" shading config

Maybe we could use `compileOnly` then? Or... idk, feels weird to have a transitive dependency here explicitly, as our shading takes care of them automatically.

Reply:
Well, we literally use databind's `ObjectMapper` directly in the `K3sContainer` class, which is why I want to explicitly declare the dependency rather than rely on it to be there by a transitive route. Ordinarily it ought to be `implementation` scope rather than `compileOnly`, but because it's already shaded in `core` I believe the only way to prevent it being added to POMs is to declare as `shaded`.

Reply:
After discussion, we've removed, which seems to work!